From 6ab7565ce1fffca0b7017b150fa66e7bd52b8554 Mon Sep 17 00:00:00 2001
From: David Mao <david.mao@amd.com>
Date: Mon, 20 Mar 2017 18:23:34 +0800
Subject: [PATCH 0794/4131] drm/amdgpu: [hybrid] fix no_evict flag

It should be merged into original patch at next rebase cycle
drm/amdgpu: expose the pinning capability to user space

Change-Id: I3eae68631825a16f4ed7912063a9005ff59f4255
Signed-off-by: David Mao <david.mao@amd.com>
Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>
Reviewed-by: David Zhou <david1.zhou@amd.com>

 Conflicts:
        drivers/gpu/drm/amd/amdgpu/amdgpu_gem.c
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_gem.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_gem.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_gem.c
index 5aba940..3ffe412b 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_gem.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_gem.c
@@ -194,7 +194,8 @@ int amdgpu_gem_create_ioctl(struct drm_device *dev, void *data,
 	if (flags & ~(AMDGPU_GEM_CREATE_CPU_ACCESS_REQUIRED |
 		      AMDGPU_GEM_CREATE_NO_CPU_ACCESS |
 		      AMDGPU_GEM_CREATE_CPU_GTT_USWC |
-		      AMDGPU_GEM_CREATE_VRAM_CLEARED))
+		      AMDGPU_GEM_CREATE_VRAM_CLEARED |
+		      AMDGPU_GEM_CREATE_NO_EVICT))
 		return -EINVAL;
 
 	/* reject invalid gem domains */
-- 
2.7.4

